rfcs/0102-moderation-team.md

```diff
@@ -0,0 +1,92 @@
+---
+feature: moderation team
+start-date: 2021-08-18
+author: tomberek
+co-authors: blaggacao
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: https://github.com/NixOS/rfcs/pull/98
+---
+
+# Summary
+[summary]: #summary
+
+Establish a team to perform moderation.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not
+sustainable to have to call on Graham any time there's a spammer or conflict
+that requires moderation, and we'd like to help the community become more
+self-regulating.
+
+(adopted from #98)
+
```

Comment:
One of the biggest issues is that the current process is not documented. It should be clear what the boundaries of the acceptable behaviors are, and what happens if they get crossed. If I get banned, how long will it be? Is there an appeal process I can use?

Moderation is a necessary chore and we should be removing as much uncertainty as possible in the process.

Reply:
I would expect a robust RFC to establish those things. The current form is pretty disappointing as it doesn't even begin to attempt to do those things.

Reply:
I would say that while the procedure for transparency and scope of appeals could be addressed here, the exact expactations might as well be a separate RFC; administrative code and code of administrative procedure are distinct in many jurisdictions.

Reply:
@zimbatm I'd like to remove the uncertainty, but I'm not sure of how to do that without opening a Pandora's box of bike-shedding and disagreement. Trying to over-prescribe exactly what the procedures are removes the human element. Additionally, trying solve the entire problem of moderation in one fell swoop does not seem wise to me. We can add something along the lines of?

"The team's composition, contact information, **guidelines, procedures**, and announcements should be maintained at https://nixos.org/community/teams/moderation.html."

This would establish an expectation that the current state is disclosed without trying to prescribe what that state should be. If something is problematic, ongoing RFCs and community discussion can address it.

If that meets the need or there is some nice self-contained/non-controversial addition, then I'd love to include it.

(yes, i AM attempting to be as minimal as possible in order to have a non-controversial RFC so we can make progress. I believe RFC98 has had enough opposition and strong opinions on either side that makes accepting it difficult at best, and divisive at worst. The discussion degraded several times to the point of being unproductive and the whole affair has become a mess.)

Reply:
In addition to the above said, I can only add that I genuinely _trust_ the team more than the current RFC authors here AND there to come up with practical guardrails.

Reply:
With regard to the assertion that "the discussion degraded several times to the point of being unproductive", I don't actually feel that that's true. As one of the authors of RFC 98 I have done my best to welcome all criticism, including criticism from the proponents of RFC 102. The proposed standard seems to be that if anyone, at any point during the discussion of an RFC, says something rude and overly personal, that this means the discussion has degraded. I hope it's clear to everyone that this would give a de facto veto to anyone who is willing to behave rudely.

I also hope everyone will reject the idea that RFC 98 is somehow unacceptable simply because there are people opposed to it. There will be people opposed to any proposal that has real substance.

Finally, I want to say that this isn't a "two sides" thing. Whatever gets enacted as part of this discussion will be for the benefit of the *entire* Nix community. I hope everyone keeps that in mind.